### ISSUE TITLE
Electron throws error randomly in CI

### ISSUE THREAD

**Issue by chrisbreiding** [cypress-io/cypress#705]
Electron will occasionally throw the following error in CI, failing the test run.

```
Error: Object has been destroyed
    at setFrameRate (/home/person/cypress-monorepo/packages/server/lib/gui/windows.coffee:192:15)
    at WebContents.<anonymous> (/home/person/cypress-monorepo/packages/server/lib/gui/windows.coffee:197:11)
    at emitThree (events.js:116:13)
    at WebContents.emit (events.js:197:7)
```

**Comment by brian-mann** [comment]
The code for this is merged, but it has not been released yet.

**Comment by brian-mann** [comment]
Fixed in [`0.20.2`](https://docs.cypress.io/guides/references/changelog.html#0-20-2).
